### ISSUE TITLE
Stats are overflowing

### ISSUE THREAD

**Issue by jwbensley** [jwbensley/EtherateMT#2]
When using a frame size of 1302 or higher (`-f 1302`) the following output is displayed:

```
1.      0.00 Rx Gbps (0 fps)    4.57 Tx Gbps (438900 fps)
2.      0.00 Rx Gbps (0 fps)    5.93 Tx Gbps (569275 fps)
3.      0.00 Rx Gbps (0 fps)    5.96 Tx Gbps (572509 fps)
4.      0.00 Rx Gbps (0 fps)    5.94 Tx Gbps (569818 fps)
```

When using 1301 or smalling:

```
1.      0.00 Rx Gbps (0 fps)    0.15 Tx Gbps (5317086108870940738 fps)
2.      0.00 Rx Gbps (0 fps)    0.00 Tx Gbps (14675157660483763884 fps)
3.      0.00 Rx Gbps (0 fps)    18446744073.71 Tx Gbps (14831125519677311053 fps)
4.      0.00 Rx Gbps (0 fps)    18446744073.71 Tx Gbps (15001272275161180849 fps)
5.      0.00 Rx Gbps (0 fps)    18446744073.71 Tx Gbps (14774409934516021121 fps)
6.      0.00 Rx Gbps (0 fps)    18446744073.71 Tx Gbps (14788588830806343604 fps)
```

**Comment by jwbensley** [comment]
Check return values aren't negative...

https://github.com/jwbensley/EtherateMT/blob/master/packet_mmap.c#L656

**Comment by jwbensley** [comment]
Fixed in Beta 0.6.
